src/cpp/ext/filters/census/client_filter.cc

```diff
@@ -56,11 +56,25 @@ grpc_error_handle CensusClientCallData::Init(
 // OpenCensusCallTracer::OpenCensusCallAttemptTracer
 //
 
+OpenCensusCallTracer::OpenCensusCallAttemptTracer::OpenCensusCallAttemptTracer(
+    OpenCensusCallTracer* parent, uint64_t attempt_num,
+    bool is_transparent_retry, bool arena_allocated)
+    : parent_(parent),
+      arena_allocated_(arena_allocated),
+      context_(CensusContext(absl::StrCat("Attempt.", parent_->method_),
```

Comment:
I don't think you need the `CensusContext()` here -- the parameters will automatically be passed to the ctor of the type of the field you're initializing.

Reply:
fixed